src/main/java/org/folio/dew/controller/BulkEditController.java

```diff
@@ -482,10 +476,10 @@ private String buildPreviewQueryFromCsv(JobCommand jobCommand, int limit) {
     try {
       Reader inputReader;
       var minioFileName = PREVIEW_PREFIX + FilenameUtils.getName(fileName);
-      if (Files.notExists(Path.of(fileName)) && repository.containsFile(minioFileName)) {
-        inputReader = new InputStreamReader(repository.getObject(minioFileName));
+      if (localFilesStorage.notExists(fileName) && remoteFilesStorage.containsFile(minioFileName)) {
+        inputReader = new InputStreamReader(remoteFilesStorage.newInputStream(minioFileName));
```

Comment:
You are using try with resources in this place: https://github.com/folio-org/mod-data-export-worker/pull/309/files#diff-39c0f2fcfdeae51e6f2d49edeb58949c937f4026f59e7c6f34ce55b453dabfd9R278
Do you need to use the same here?

Reply:
This inputReader is used below in the CSVReader that is in the try-with-resources. So I assumed it should be closed automatically together with CSVReader. But JIC I'll introduce it in the try-with-resources. Thanks.